src/views/AuthorityView/AuthorityView.js

```diff
@@ -46,23 +39,28 @@ const propTypes = {
   }).isRequired,
 };
 
-const AuthorityView = ({
-  marcSource,
-  authority,
-}) => {
+const AuthorityView = ({ marcSource, authority }) => {
+  const [requestDeletion, setRequestDeletion] = useState(false);
   const intl = useIntl();
   const history = useHistory();
   const location = useLocation();
   const stripes = useStripes();
 
-  const [, setSelectedAuthorityRecordContext] = useContext(SelectedAuthorityRecordContext);
+  const [, setSelectedAuthorityRecordContext] = useContext(
+    SelectedAuthorityRecordContext,
```

Comment:
We only format calls with parameters on each line where there are too many parameters to put in a single line. ESLint should give an error when that happens